GOVERNANCE.md

```diff
@@ -9,49 +9,72 @@ In order to provide equitable rights to all Working Group members,
 the following process will be followed:
 
 * Official WG meetings will be announced at least a week in advance.
-* Proposed changes to any document will be done via a Pull Request (PR).
-* PRs will be reviewed during official WG meetings.
+* Proposed changes to any document will be done via a Pull Request (PR), 
+  see below for policy on review and acceptance of PRs.
 * During meetings, priority will be given to PRs that appear to be ready for
   a vote over those that appear to require discussions.
-* PRs should not be merged if they have had substantial changes made within
-  two days of the meeting.
-  Rebases, typo fixes, etc. do not count as substantial.
-  Note, administrivia PRs that do not materially modify WG output documents
-  may be processed by WG admins as needed.
-* Resolving PRs ("merging" or "closing with no action") will be done as a
-  result of a motion made during a WG meeting, and approved.
-* Reopening a PR can be done if new information is made available, and a
-  motion to do so is approved.
 * Any motion that does not have "unanimous consent" will result in a formal
   vote. See [Voting](#voting).
 
+## Working Group Membership
+
+Anyone is welcome to attend Working Group meetings and invited to review 
+PRs, whether or not they have declared formal membership in the Working Group.
+
+We encourage everyone to document your participation in the [member list](), 
+then people new to the group can learn about other folks who are involved.
+Each member identifies their affiliation with an organization, company or
+open source project, along with their specific use case(s) for events.
+
+- company / org / project 
+    - representative, (optional) alternate representative
+    - describe how you are using or intend to use the Event specification
+    
+**Active Members** participate in at least three Working Group meetings per month.
```

Comment:
Is there an alternative mechanism for members to participate in the Working Group? (e.g. comment on meeting notes, add material agenda items?) It would be fine for e.g. each meeting to approve previous minutes plus a list of the last week's non-attending participants.

I've had the meeting on my list to drop by, but I have an external scheduling conflict (preschool), which makes Thursdays difficult to attend.

Reply:
I also will have trouble attending, but I'd like to be able to make async time to participate. Maybe there's some amount of contribution we can do in PRs or something.

Reply:
I would add that this requirement is rather geographically limiting. Perhaps active async reviews of PRs prior to meetings could be counted as participation for the week?

Reply:
We talked about how we'll eventually need to move to a more traditional model of maintainers being able to approve PRs outside of our calls. And I think that'll address that concern - and speed up our calls :-)

I would suggest that people use this issue: https://github.com/cloudevents/spec/issues/21  to gather ideas on how we migrate.